Link to fingerprinting grammar (#8094)
getsentry/sentry-docs: src/docs/product/data-management-settings/event-grouping/fingerprint-rules.mdx

1. Identify the match logic for grouping issues together.
1. Set the match logic and the fingerprint for it.

> The syntax for fingerprint rules is similar to that of [**Discover** queries](/product/sentry-basics/search/#syntax). If you want to negate the match, prefix the expression with an exclamation mark (`!`). The full grammar is defined [here](https://github.com/getsentry/sentry/blob/c5b84a393365a833348dd7fd9378d34c353ca6ca/src/sentry/grouping/fingerprinting.py#L17-L55).

Reviewer: ```suggestion
The syntax for fingerprint rules is similar to syntax in [**Discover** queries](/product/sentry-basics/search/#syntax). To negate a match, add an exclamation mark (`!`) at the beginning of the expression. See the full list of grammar [here](https://github.com/getsentry/sentry/blob/c5b84a393365a833348dd7fd9378d34c353ca6ca/src/sentry/grouping/fingerprinting.py#L17-L55).
```

Author: Thanks! `list of grammar` did not sound right to me, so I changed it to "See the full grammar here".

> The syntax for fingerprint rules is similar to that of [**Discover** queries](/product/sentry-basics/search/#syntax). If you want to negate the match, prefix the expression with an exclamation mark (`!`). The full grammar is defined [here](https://github.com/getsentry/sentry/blob/c5b84a393365a833348dd7fd9378d34c353ca6ca/src/sentry/grouping/fingerprinting.py#L17-L55).

Author: ```suggestion
The syntax for fingerprint rules is similar to syntax in [**Discover** queries](/product/sentry-basics/search/#syntax). To negate a match, add an exclamation mark (`!`) at the beginning of the expression. See the full grammar [here](https://github.com/getsentry/sentry/blob/c5b84a393365a833348dd7fd9378d34c353ca6ca/src/sentry/grouping/fingerprinting.py#L17-L55).
```
